docs/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+## what
```

Comment:
Wrong location for this template

Reply:
I guess `docs/` works to, but lets standardize on `.github/PULL_REQUEST_TEMPLATE/
`

Reply:
If you create a `.github/PULL_REQUEST_TEMPLATE/` directory, then (my understanding is) you no longer have a default template, but instead have to specify which template to use via a query parameter on the pull request URL. I don't like that. 

Really, though, we should be creating [org-wide default templates](https://help.github.com/en/articles/creating-a-default-community-health-file-for-your-organization) in https://github.com/cloudposse/.github